templates/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml.in

```diff
@@ -1,9 +1,9 @@
 packageName: kubevirt-hyperconverged
 channels:
-- name: "0.0.3"
-  currentCSV: kubevirt-hyperconverged-operator.v0.0.3
+- name: "1.0.0"
+  currentCSV: kubevirt-hyperconverged-operator.v1.0.0
```

Comment:
Are you renaming 0.0.3 to 1.0.0? Or adding 1.0.0?

Reply:
Adding 1.0.0